hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el

```diff
@@ -0,0 +1,87 @@
+INSTANCE:tb.dut.u_tlul_adapter_sram.u_tlul_data_integ_enc_data.u_data_gen
+
+CHECKSUM: "2503688253 3812101441"
+INSTANCE: tb.dut.u_reg_regs.u_reg_if.u_rsp_intg_gen
+ANNOTATION: "[UNR] all inputs are constrant"
+Block 1 "461445014" "assign rsp_intg = tl_i.d_user.rsp_intg;"
+ANNOTATION: "[UNR] all inputs are constrant"
+Block 2 "2643129081" "assign data_intg = tl_i.d_user.data_intg;"
+
+CHECKSUM: "281300783 1779845869"
+INSTANCE: tb.dut.u_reg_regs.u_status_bus_integ_error.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constrant"
+Block 2 "1620753216" "assign wr_data = d;"
+INSTANCE: tb.dut.u_reg_regs.u_status_init_error.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constrant"
+Block 2 "1620753216" "assign wr_data = d;"
+
+CHECKSUM: "281300783 1779845869"
+INSTANCE: tb.dut.u_reg_regs.u_status_escalated.wr_en_data_arb
+ANNOTATION: "[UNR] all inputs are constrant"
+Block 2 "1620753216" "assign wr_data = d;"
+
+CHECKSUM: "1738953883 3087984712"
+INSTANCE: tb.dut.u_reg_regs.u_chk
+ANNOTATION: "[LOWRISCK] when a_valid is low, we drive X on data, so this condition isn't covered"
+Condition 1 "899198529" "(tl_i.a_valid & (((|err)) | ((|data_err)))) 1 -1" (1 "01")
+
+CHECKSUM: "1296247128 1854270750"
+INSTANCE: tb.dut
+ANNOTATION: "[UNSUPPORTED] ACK can't come without REQ"
+Condition 18 "153333633" "(key_ack & ((~key_req)) & ((~local_esc))) 1 -1" (2 "101")
+ANNOTATION: "[LOWRISCK] we don't issue a new init when there is a unfinished init"
+Condition 11 "1796319142" "(init_done & ((~init_trig)) & ((~local_esc))) 1 -1" (2 "101")
+
+CHECKSUM: "704952876 1147758610"
+INSTANCE: tb.dut.u_prim_sync_reqack_data.u_prim_sync_reqack
+ANNOTATION: "[UNSUPPORTED] ACK can't come without REQ"
+Condition 1 "1414883863" "(src_req_i & src_ack_o) 1 -1" (1 "01")
+
+CHECKSUM: "2574923469 2212102968"
+INSTANCE: tb.dut.u_tlul_adapter_sram.u_sram_byte
+ANNOTATION: "[UNR] shouldn't happen, read is finished in 1 cycle"
+Condition 2 "3441602297" "(gen_integ_handling.sram_d_ack && gen_integ_handling.rd_wait) 1 -1" (1 "01")
+
+CHECKSUM: "835220981 1390693035"
+INSTANCE: tb.dut.u_tlul_adapter_sram.u_rspfifo
+ANNOTATION: "[UNR] when rspfifo is full, we don't expect to receive a request, as it's blocked at the req phase"
+Condition 4 "786039886" "(wvalid_i & wready_o & ((~gen_normal_fifo.under_rst))) 1 -1" (2 "101")
+ANNOTATION: "[LOWRISCK] we don't issue a req when it's under reset"
+Condition 7 "1709501387" "(((~gen_normal_fifo.empty)) & ((~gen_normal_fifo.under_rst))) 1 -1" (2 "10")
+
+CHECKSUM: "835220981 869192578"
+INSTANCE: tb.dut.u_tlul_adapter_sram.u_sramreqfifo
+ANNOTATION: "[UNR] this fifo can never be full"
+Condition 1 "644960181" "(gen_normal_fifo.full ? (2'(Depth)) : ((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((2'(gen_normal_fifo.wptr_value) - 2'(gen_normal_fifo.rptr_value))) : (((2'(Depth) - 2'(gen_normal_fifo.rptr_value)) + 2'(gen_normal_fifo.wptr_value))))) 1 -1" (2 "1")
+ANNOTATION: "[UNR] because this fifo can never be full"
```

Comment:
`[UNR] this fifo can never be full, because transactions can drain into u_rspfifo`